src/Persistence/ProxyGenerator.php

```diff
@@ -114,12 +114,23 @@ private static function generateClassFor(object $object): string
             $proxyCode,
             [
                 'implements \Symfony\Component\VarExporter\LazyObjectInterface' => \sprintf('implements \%s, \Symfony\Component\VarExporter\LazyObjectInterface', Proxy::class),
+                'use \Symfony\Component\VarExporter\Internal\LazyDecoratorTrait' => \sprintf("use \\%s;\n    use \\%s", IsProxy::class, LazyProxyTrait::class),
                 'use \Symfony\Component\VarExporter\LazyProxyTrait' => \sprintf("use \\%s;\n    use \\%s", IsProxy::class, LazyProxyTrait::class),
-                'if (isset($this->lazyObjectState)) {' => "\$this->_autoRefresh();\n\n        if (isset(\$this->lazyObjectReal)) {",
                 '\func_get_args()' => '$this->unproxyArgs(\func_get_args())',
             ],
         );
 
+        /**
+         * Add `$this->_autoRefresh();` after every method declaration.
+         */
+        $proxyCode = preg_replace_callback(
+            '/^(\s*(?:public|protected|private)?\s*(?:static\s+|final\s+|abstract\s+)*function\s+(?!__)\w+\s*\([^\)]*\)\s*):?\s*\??[\w\\\\]*\s*\{\s*$/m',
+            function ($matches) {
+                return rtrim($matches[0]) . "\n    \$this->_autoRefresh();";
+            },
+            $proxyCode
+        );
```

Comment:
I had to rework how we introduce the call to `_autoRefresh()` 

`symfony/var-exporter` used to generate code like this for any method:
```php
    public function getContact(): ?\Zenstruck\Foundry\Tests\Fixture\Entity\Contact
    {
        if (isset($this->lazyObjectReal)) {
            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->getContact(...$this->unproxyArgs(\func_get_args()));
        }

        return parent::getContact(...$this->unproxyArgs(\func_get_args()));
    }
```
so we were relying on the `if()` to introduce the call. But now, with php 8.4 + sf 7.3 (again, only for this combo), there is no more if, so I changed how it worked to use a regex which looks for any method declaration